lib/extensions/postgres_cdc_stream/replication.ex

```diff
@@ -124,7 +121,7 @@ defmodule Extensions.PostgresCdcStream.Replication do
     Logger.debug("Got message: #{inspect(msg)}")
     [{_, columns, schema, table}] = :ets.lookup(state.tid, msg.relation_id)
 
-    %NewRecord{
+    record = %NewRecord{
```

Comment:
This variable will not be used anywhere except in the next function. Did you take this from the coding style guidelines?